packages/angular_devkit/build_angular/src/angular-cli-files/models/build-options.ts

```diff
@@ -87,8 +87,8 @@ export interface BuildOptions {
 
   /* Append script target version to filename. */
   esVersionInFileName?: boolean;
-
   experimentalRollupPass?: boolean;
+  allowedCommonJsDepedencies?: string[];
```

Comment:
`allowedCommonJsDepedencies` -> `allowedCommonJsDependencies` (missing `n` in `pen`)

Reply:
Thanks!